src/templates/cashier/payment_methods.html.tt

```diff
@@ -1,64 +1,202 @@
-        </div>
+<div class="container">
+    <div class="row">
+        <div class="col-sm">
+            <div id="payment_methods" class="row">
+                <div class="col-xs-4 col-sm-3 col-md-12">
+                    <div class="row tabs-container">
+                        <div class="previous-button hide">
+                            <div class="arrow arrow-left arrow-up"></div>
+                        </div>
+                        <div class="col-md-11 tab-buttons">
+                            [% BLOCK payment_methods_tabs %]
+                                <ul class="scrollable-tabs">
+                                    [% FOREACH method IN methods %]
+                                    <li class="col-sm-12 col-md-2 [% IF method.id=='visa' %]tab-selected[% END %] ">
+                                        <a rel="#[% method.id %]"><div class="tab-icon [% method.id %]"></div></a>
+                                    </li>
+                                    [% END %]
+                                </ul>
+                            [% END %]
+                            [% INCLUDE payment_methods_tabs %]
+                        </div>
+                        <div class="next-button col-md-1">
+                            <div class="arrow arrow-right arrow-down"></div>
+                        </div>
+                    </div>
+                </div>
 
-        [% BLOCK row %]
-            <tr class="gr-row gr-padding-10 [% is_header ? 'table-header' : 'table-body' %]">
-                <td>[% IF id %]<div id="[% id %]" class="payment-logo"></div>[% ELSE %][% method %][% END %]</td>
-                <td>[% currencies %]</td>
-                <td>[% deposit.join('<br />') %]</td>
-                <td>[% withdrawal.join('<br />') %]</td>
-                <td>[% time.join('<br />') %][% is_header ? '*' : '' %]</td>
-            </tr>
-        [% END %]
-        [% BLOCK section %]
-            <h3>[% title %]</h3>
-            <div class="table-container">
-                <table>
-                    <thead>
-                        [% INCLUDE row is_header=1 method=l('Method') currencies=l('Currencies') deposit=l('Min-Max Deposit') withdrawal=l('Min-Max Withdrawal') time=l('Processing Time') %]
-                    </thead>
-                    <tbody>[% content %]</tbody>
-                </table>
-            </div>
-        [% END %]
+                <div class="tab-content col-xs-8 col-sm-9 col-md-12">
+                    [% BLOCK tab_content %]
+                        [% FOREACH method IN methods %]
+                        <div id=[% method.id %] class="row">
+                            <div class="col-sm-12 col-md-3">
+                                <p>[% l('Currencies') %]</p>
+                                <h4>[% method.currencies.join(', ') %]</h4>
+                                <p>[% l('Processing time*') %]</p>
+                                <h4>[% l('Deposit: ') %][% method.deposit.process_time %]
```

Comment:
Better to prevent translators from accidentally removing necessary space at the end.